exercise-1.js

```diff
@@ -444,74 +711,148 @@
         */
 
         //your code here
-
-
+        function printRole() {
+            for (i=0; i<devs.length; i++){
+                let name = devs[i].name;
+                let tech = devs[i].tech_stack;
+                console.log(`${name} specializes is ${tech}.`);
+            }
+        }
+        printRole();
         /************************************************************* */
         // Write a function to find the maximum numerical value of the given array.  Get rid of any non numerical values.  Convert the strings that are numbers to an actual number data type.  ("one" => 1) ("1" => 1).  Use array methods to perform this task.  
         const numbersMixed = [2,23,1,2,1,1,1,2,2.5,20,200,2000,,{k:"val"},20000,19999,1878,140,23,4,"sk",true,true,"true-dat","nice","one","two","three","3","tea",[]];
 
         function maxNumber(numbers) {
-            //your code...
+            let onlyNumbers = [];
+            //console.log(numbers);
+            for (i=0; i < numbers.length; i++) {
+                if (typeof numbers[i] == 'number') {
+                    onlyNumbers.push(numbers[i]);
+                } else if (numbers[i] === "one") {
+                    onlyNumbers.push(1);
+                } else if (numbers[i] === "two") {
+                    onlyNumbers.push(2);
+                } else if (numbers[i] === "three") {
+                    onlyNumbers.push(3);
+                } else if (numbers[i] === "3"){
+                    onlyNumbers.push(3);                    
+                }
+            }
+            return onlyNumbers;
         }
+        let justNumbers = maxNumber(numbersMixed); // NEED TO FINISH
+        console.log(Math.max(...justNumbers));
 
         //After the numbers array has been cleaned up to only have numbers in it, Write a function that sorts the modified numbers array.  Allow the function to sort the array in descending order as well.
 
         function sortNums(numbers,desc=false) {
```

Comment:
This function fails to meet the requirements.  Currently, whether the boolean is set to 'false' or 'true', the numbers are sorted in descending order.